src/makeRobotAccountant.js

```diff
@@ -6,7 +6,26 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
-}
+  let sum = 0;
+  let isEven = 0;
+
+  return function getSum(num1) {
+    // debugger;
+    isEven++;
+    sum++;
+
+    if (isEven > 3 && isEven % 2 === 0) {
```

Comment:
```suggestion
    if (sum > 3 && sum % 2 === 0) {
```
In this case you don't need `isEven`